ee/debug/checkups/desktop_menu.go

```diff
@@ -0,0 +1,85 @@
+package checkups
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/kolide/launcher/ee/agent/types"
+)
+
+type desktopMenu struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (d *desktopMenu) Name() string {
+	return "Desktop Menu"
+}
+
+func (d *desktopMenu) Run(_ context.Context, fullFH io.Writer) error {
+	menuJsonPath := filepath.Join(d.k.RootDirectory(), "menu.json")
+
+	if _, err := os.Stat(menuJsonPath); err != nil {
+		d.status = Failing
+		d.summary = fmt.Sprintf("failed to stat menu json: %s", err)
+		return nil
+	}
+
+	file, err := os.Open(menuJsonPath)
+	if err != nil {
+		d.summary = fmt.Sprintf("failed to open %s: %s", menuJsonPath, err)
```

Comment:
do we want to set a status for any of these other error cases?

Reply:
yep, good catch, corrected ... set to failing at top, passing just before finishing